components/card.css

```diff
@@ -0,0 +1,67 @@
+
+.card-vertical {
+    display: flex;
+    flex-direction: column;
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
```

Comment:
You could create a shadow class , just a suggestion

Reply:
Good point. I've changed it.